mkosi/machine.py

```diff
@@ -31,16 +35,30 @@
 
 
 class Machine:
-    def __init__(self, args: Optional[Sequence[str]] = None, debug: bool = False) -> None:
-        # Remains None until image is built and booted, then receives pexpect process
+    def __init__(self, args: Sequence[str], debug: bool = False) -> None:
```

Comment:
Mypy, in line 51, does not allows me to look for the `Verb.build` in `args` if it's type is `Optional[Sequence[str]] = None`.
```

mkosi/machine.py:51:39: error: Unsupported right operand type for in ("Optional[Sequence[str]]")  [operator]
            if tmp.verb == Verb.build and Verb.build.name not in args:
```

Reply:
Adding `= []` to the constructor should be fine here, since `Sequence` means the variable is read-only and can't be modified. Just be careful that you can never use mutable objects as the default value as they will not be reset after the method finishes.